src/components/TextField/README.md

```diff
@@ -115,7 +115,8 @@ input. Field placeholder text should:
 ### Designating optional fields
 
 Try to only ask for information that’s required. If you need to ask merchants
-to provide optional information, mark the field optional by placing the text “(optional)” at the end of the field’s label. Don’t mark required fields with asterisks.
+to provide optional information, mark the field optional by placing the text “(optional)” at the end of the field’s label.
+Don’t mark required fields with asterisks unless it is expected by the [local cultural norm](https://polaris.shopify.com/foundations/internationalization#section-internationalize-polaris-components).
```

Comment:
Did you mean to link to the section above, [Plan for cultural differences](https://polaris.shopify.com/foundations/internationalization#plan-for-cultural-differences)?

Reply:
thanks for catch!